### PR TITLE
Feat : Authentication

### PR DESCRIPTION
**Pull request by manojkumar8400**
Added default authentication to this app.

### CODE REVIEW

File: src/components/404Page/PageNotFound.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 }
 
 .img-404 {
-    width: 70%;
-    height: 69vh;
+    width: 35rem;
+    /* height: 69vh; */
```
Comment (Chakravarthi0) [inline src/components/404Page/PageNotFound.css:10]: You can remove this comment, if not required.

File: src/pages/Auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { Link } from "react-router-dom";
+// import "./Signup.css";
```
Comment (Chakravarthi0) [inline src/pages/Auth/Signup.jsx:2]: You can remove this comment if not required

File: src/pages/HomePage/Home.jsx [hunk 3]
```diff
@@ -25,19 +25,25 @@ const HomePage = () => {
                     <img className="quiz-img radious" src="https://lh3.googleusercontent.com/DXYqwbqhBeF9FnEpridWKO6-Rj0LMlcKITlco1wz2kx9Ji5e5IvzVLp3baj926dZUDY" alt="ipl quiz image" />
                     <h2>IPL Quiz Game</h2>
                     <p className="text-align f-size-16">Check your knowledge of Indian Premiere League</p>
-                    <button className="play-btn w-100 p-8">Play Now</button>
+                    <Link to="./rule" className="w-100">
```
Comment (Chakravarthi0) [inline src/pages/HomePage/Home.jsx:28]: You dont need dot here, you can give ```to="/rule"```

File: src/pages/index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import Rules from "./RulesPage/Rules";
 import HomePage from "./HomePage/Home";
 import McqPage from "./McqPage/McqPage";
+import Login from "./Auth/Login";
+import Signup from "./Auth/Signup";
 
-export { Rules, HomePage, McqPage }
\ No newline at end of file
+export { Rules, HomePage, McqPage, Login, Signup }
```
Comment (Chakravarthi0) [inline src/pages/index.js]: You can directly eport the compoennts using barrel export

**Review by Chakravarthi0 — APPROVED** [review]
suggestion: add a preview link or a video/screenshots.

**Review by manojasarsa — APPROVED** [review]
Code looks clean,  GTG!

File: src/pages/index.js (hunk 4 above)
Comment (manojkumar8400) [reply src/pages/index.js]: I will keep these things in mind in future.

**Pull request merged**